.github/workflows/discord_pull_requests.yaml

```diff
@@ -24,5 +25,5 @@ jobs:
           destination: ${{ secrets.DISCORD_BOT_DESTINATION }}
           issue_number: ${{ github.event.inputs.pull_request_number || github.event.pull_request.number }}
           issue_comment: Hey 👋, I've just created a [thread]($THREAD_LINK$) for this pull request on [PyTorch-Ignite Discord](https://pytorch-ignite.ai/chat) where you can quickly talk to the community on the topic.
-          discord_message: New PR created in `${{ github.repository }}`:<https://github.com/${{ github.repository }}/pull/${{ github.event.pull_request.number }}>
+          discord_message: New PR created in `${{ github.repository }}`:<https://github.com/${{ github.repository }}/pull/${{ github.event.pull_request.number || github.event.pull_request.number }}>
```

Comment:
`github.event.pull_request.number || github.event.pull_request.number` ??

Reply:
@vfdev-5 Since both values mostly give a same output it is better to have this structure as I am a bit unsure about the Github API requests in the actions. It is similar to fetching of the `issue_number: ${{ github.event.inputs.pull_request_number || github.event.pull_request.number }}` in line 26

Reply:
Correct me if I'm wrong but I see the same variable twice, like `${{ x || x}}`. If this is expected, I do not understand why can't you do `${{ x }}` ?